content/en/docs/concepts/configuration/assign-pod-node.md

```diff
@@ -362,6 +362,42 @@ For more information on inter-pod affinity/anti-affinity, see the
 You may want to check [Taints](/docs/concepts/configuration/taint-and-toleration/)
 as well, which allow a *node* to *repel* a set of pods.
 
+## nodeName
+
+`nodeName` is the simplest form of node selection constraint, but due
+to its limitations its use is _not_ recommended.  `nodeName` is a
+field of PodSpec.  If it is non-empty, the scheduler ignores the pod
+and the kubelet running on the named node tries to run the pod.  Thus,
+if `nodeName` is provided in the PodSpec, it takes precedence over the
+above methods for node selection since satisfying their constraints is
```

Comment:
Please remove the second part of this sentence: "since satisfying ...".

Reply:
Removed